unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentView.cs

```diff
@@ -152,12 +156,30 @@ private void OpenCurrentWalletSection()
         SelectSection(ExploreSection.Wallet, true);
     }
 
+    private void OpenMyAccountSection(bool current, bool previous)
+    {
+        if (!current)
+            return;
+
+        foreach (var section in exploreSectionsById)
```

Comment:
We can extract this foreach in a function to provide a meaningful name and reuse it in other parts of the code like in `OpenCurrentWalletSection`